include/quic/gnutls_crypto.hpp

```diff
@@ -2,20 +2,97 @@
 
 extern "C"
 {
+#include <gnutls/abstract.h>
 #include <gnutls/crypto.h>
 #include <gnutls/gnutls.h>
 #include <ngtcp2/ngtcp2.h>
 #include <ngtcp2/ngtcp2_crypto.h>
 #include <ngtcp2/ngtcp2_crypto_gnutls.h>
 }
 
+#include <oxenc/base64.h>
+#include <oxenc/hex.h>
+
 #include <optional>
 #include <variant>
 
 #include "crypto.hpp"
+#include "utils.hpp"
 
 namespace oxen::quic
 {
+    class Connection;
+
+    inline std::string translate_key_format(gnutls_x509_crt_fmt_t crt)
```

Comment:
This could be a `inline constexpr std::string_view` return type

Reply:
(The two functions below this, as well)

Reply:
Narrator: "No they couldn't..."

[here](https://ci.oxen.rocks/dr7ana/libquic/290/2/3)